ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java

```diff
@@ -183,6 +184,7 @@ private static void usage() {
         final Supplier<InputStream> schemaFile,
         final Format keyFormat,
         final Format valueFormat,
+        final Character valueDelimiter,
```

Comment:
Can we make this:?

```suggestion
        final char valueDelimiter,
```

Reply:
Again, I think it's valid for value_delimiter to be null;

Reply:
As above.

Reply:
Should be handled by DataGen refactoring